vector/src/test/scala/spec/geotrellis/vector/reproject/ReprojectSpec.scala

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  */
 
-package geotrellis.vector
+package geotrellis.vector.reproject
```

Comment:
I think we need to review the vector package specs structure in a separate issue @echeipesh. 
Test paths here don't correspond to package names.